src/test/java/com/conveyal/gtfs/GTFSFeedTest.java

```diff
@@ -26,37 +26,85 @@ public class GTFSFeedTest {
 
     private static final Logger LOG = LoggerFactory.getLogger(GTFSFeedTest.class);
     private static String simpleGtfsZipFileName;
+    private static String simpleFlexGtfsZipFileName;
 
     @BeforeAll
     public static void setUpClass() {
         //executed only once, before the first test
         simpleGtfsZipFileName = null;
+        simpleFlexGtfsZipFileName = null;
         try {
             simpleGtfsZipFileName = TestUtils.zipFolderFiles("fake-agency", true);
+            simpleFlexGtfsZipFileName = TestUtils.zipFolderFiles("fake-agency-with-flex", true);
         } catch (IOException e) {
             e.printStackTrace();
         }
     }
 
     /**
-     * Make sure a roundtrip of loading a GTFS zip file and then writing another zip file can be performed.
+     * Make sure a round-trip of loading a GTFS zip file and then writing another zip file can be performed.
      */
     @Test
     public void canDoRoundtripLoadAndWriteToZipFile() throws IOException {
-        // create a temp file for this test
-        File outZip = File.createTempFile("fake-agency-output", ".zip");
-
-        // delete file to make sure we can assert that this program created the file
-        outZip.delete();
-
-        GTFSFeed feed = GTFSFeed.fromFile(simpleGtfsZipFileName);
-        feed.toFile(outZip.getAbsolutePath());
-        feed.close();
-        assertThat(outZip.exists(), is(true));
+        FileTestCase[] fileTestCases = {
+            // agency.txt
```

Comment:
I think we can remove this comment if the file name is in the `FileTestCase` directly ?